CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Added
 
 - `var` library function for reading environmental variables
+- Support for `u8`, `u16`, `u32`, `u64`, `u128`, `u256` types to fuzzer
```

Comment:
Can we add support for i8, i16 etc. as well?

Reply:
I think we can do that, but I'd do it in a separate PR, since it will require some minimal changes to the logic so we handle negative values.

Reply:
#755